docs/tutorials/10_effective_dimension.ipynb

```diff
@@ -0,0 +1,763 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "num_inputs = 2\n",
+    "num_samples = 20\n",
+    "\n",
+    "X = 2 * algorithm_globals.random.random([num_samples, num_inputs]) - 1\n",
+    "y01 = 1 * (np.sum(X, axis=1) >= 0)  # in { 0,  1}\n",
+    "y = 2 * y01 - 1  # in {-1, +1}\n",
+    "\n",
+    "for x, y_target in zip(X, y):\n",
+    "    if y_target == 1:\n",
+    "        plt.plot(x[0], x[1], \"bo\")\n",
+    "    else:\n",
+    "        plt.plot(x[0], x[1], \"go\")\n",
+    "plt.plot([-1, 1], [1, -1], \"--\", color=\"black\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The next step is to create a QNN, an instance of `TwoLayerQNN` in our case. Since we pass only the number of inputs, the network will continue with the default values for feature map and ansatz."
```

Comment:
I guess this is done around TwoLayerQNN which maybe is why the former above is CircuitQNN to show that can be done too?

Reply:
Yes, this was the idea, to have examples with both.